packages/components/src/components/primitives/TextPairing/TextPairing.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { TextPairing, TextPairingVariant, textPairingVariants } from '.';
+import { Text } from 'components';
+import { Box, Flex, Spacer, Stack, StackDivider } from '@chakra-ui/layout';
+
+export default {
+  component: TextPairing,
+  title: 'Components/Primitives/TextPairing',
+  argTypes: {
+    variant: {
+      options: textPairingVariants,
+      control: { type: 'select' },
+    },
+  },
+} as Meta;
+
+interface Props {
```

Comment:
Could we use the existing type from TextPairing?

Reply:
done